src/main/java/Daisy.java

```diff
@@ -0,0 +1,208 @@
+import daisy.error.IllegalEntryException;
+import daisy.error.IllegalEventFormatException;
+import daisy.error.IllegalDeadlineFormatException;
+import daisy.task.Deadline;
+import daisy.task.Event;
+import daisy.task.Task;
+import daisy.task.Todo;
+
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.FileWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+
+public class Daisy {
+
+    protected static ArrayList<Task> tasks = new ArrayList<>();
+
+    protected static String defaultFileLocation = System.getProperty("user.dir") + "\\src\\main\\java\\daisy\\data\\Daisy.txt";
+
+    protected static String INTRO_PROMPT = "Good day! This is Daisy.\nAny task for today?";
+    protected final static String EXIT_PROMPT = "Ending prompt received. Remember to keep to the deadlines!";
+    protected final static String LINE_BREAK = "____________________________________";
+
+    public static void main(String[] args) {
+
+        loadData();
+
+        System.out.println(LINE_BREAK);
+        System.out.println(INTRO_PROMPT);
+        System.out.println(LINE_BREAK);
+
+        Scanner in = new Scanner(System.in);
+        String command = in.nextLine();
+
+        while (!command.equals("bye")) {
+            System.out.println(LINE_BREAK);
+            String[] separate_commands = command.split(" ",2);
+            switch (separate_commands[0]) {
+                case "list":
+                    for (Task task : tasks) {
+                        System.out.println((tasks.indexOf(task) + 1) + "." + task);
+                    }
+                    break;
+                case "mark":
+                    tasks.get(Integer.parseInt(separate_commands[1])-1).setDone();
+                    System.out.println("Congrats on completing the task!");
+                    System.out.println(tasks.get(Integer.parseInt(separate_commands[1])-1));
+                    break;
+                case "unmark":
```

Comment:
Can define all constants at the top for all command strings to prevent using magic literals